### ISSUE TITLE
Add Rock and Roll Cowboy to the shows section

### ISSUE THREAD

**Issue by BoscoBonetti** [kvcmlive/website#29]
Content emailed.
---
title: "Title"
excerpt: "" # leave this one empty
header:
  teaser: /assets/images/ImageName
sidebar:
  - image: /assets/images/ImageName
  - title: "TimeSlot"
  - title: "Genres"
    text: "GenreList"
---

Show Description
```

**Add any further details here:**
e.g. social media links, playback links, show creator, etc.

**Comment by oofdere** [comment]
Please fill in the template in the future.

**Comment by oofdere** [comment]
closed by 942f823
